src/client/pages/EventPage/components/AppsOverTimeStatistics.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { VictoryChart, VictoryAxis, VictoryLine, VictoryTheme, VictoryVoronoiContainer, VictoryTooltip } from 'victory';
+
+import EventStatisticsComponent from './EventStatisticsComponent';
+
+export default class AppsOverTimeStatistics extends EventStatisticsComponent {
+  render() {
+    const { statistics } = this.props;
+
+    // Create the data array needed to make the line chart
+    const appsTimeData: Array<{ date: string; appCount: number }> = [];
+
+    // Push stats query data to array of data points
+    for (const [key, value] of Object.entries(statistics.appsOverTime)) {
+      appsTimeData.push({
+        date: key,
+        appCount: value,
+      });
+    }
+
+    // Lastest applicants vs time data
+    const latest = appsTimeData[appsTimeData.length - 1];
+
+    const axisStyles = {
+      tickLabels: { fontSize: 5, padding: 5 },
+      ticks: { size: 2, stroke: '#90A4AE'},
+      grid: { opacity: 0 },
+      axisLabel: { fontSize: 8, padding: 20 },
+    };
+
+    return (
+      <div className="event-statistics event-page__card">
+        <h2>Applicants Over Time</h2>
+        <div className="row">
+          <div className="col-12">
+            <VictoryChart
+              theme={VictoryTheme.material}
+              minDomain={{ y: 0 }}
+              maxDomain={{ y: latest.appCount + 3, x: appsTimeData.length + 1 }}
```

Comment:
bug: App crashes when rendering statistics for an event with 0 users

`appsTimeData` is an empty array and `latest = appsTimeData[length - 1]` is `undefined` (lol js) meaning `.appCount` is an access on undefined memory. 

intended: render empty graph?

Reply:
Yes I think it should render an empty graph. Right now the empty graph looks weird though: the x-axis are showing decimals instead of dates. I think ima make a separate small PR to make edge cases more elegant, and I also put handling edge cases as another task in issue #177